examples/mgui_with_threading.py

```diff
@@ -1,6 +1,13 @@
 """An example of calling a threaded function from a magicgui dock_widget.
 Note: this example requires python >= 3.9
 """
+
+import sys
+
+if sys.version_info < (3, 9):
```

Comment:
Wow, I had no idea that's how to compare the version!

Reply:
Neat, right? :)

Reply:
It's "new" in 3+, in 2.x you actually had to slice.